redis.go

```diff
@@ -60,6 +60,12 @@ func (ds *Datastore) Get(key datastore.Key) (value []byte, err error) {
 	return ds.client.Cmd("GET", key.String()).Bytes()
 }
 
+func (ds *Datastore) GetSize(key datastore.Key) (size int, err error) {
+	ds.mu.Lock()
+	defer ds.mu.Unlock()
+	return ds.client.Cmd("STRLEN", key.String()).Int()
```

Comment:
From what I'm reading [`string`](https://redis.io/topics/data-types) in Redis means just arbitrary binary data so this function is safe, right?

Reply:
It should be (although I'm unfamiliar with rRdis). I just want the number of bytes in the block which should correspond to STRLEN.